bld/namelist_files/namelist_definition_pop.xml

```diff
@@ -2250,15 +2250,14 @@ Flag for using CVMix for mixing instead of POP routines
 Default: .true.
 </entry>
 
-<!--QL, 150526, Langmuir mixing options-->
 <entry
 id="langmuir_opt"
 type="char*256"
 category="Vertical Mixing (KPP)"
 group="vmix_kpp_nml" >
```

Comment:
The `Valid Values` section at line 2260 is just there for documentation purposes. if you edit line 2257 to add the `valid_values` attribute to the `entry` then `build-namelist` will abort if it comes across a valid not in the list:

```
- group="vmix_kpp_nml" >
+ group="vmix_kpp_nml"
+ valid_values="null,vr12-ma,lf17" >
```

Reply:
OK. Sorry I missed that... Now it should be fixed.